Framework/Mobile/LibTest/AppDelegate.m

```diff
@@ -1,41 +0,0 @@
-#import "AppDelegate.h"
-#import "ViewController.h"
-
-#import <CocoaLumberjack/CocoaLumberjack.h>
-#import "Formatter.h"
```

Comment:
I actually remade this target and a few of the others inside the framework folder. I used the same project for both OS X & iOS and set it on a per target basis. I'm not sure about removing them, as they do serve to check that the dynamic frameworks & static libs can be built and integrated into a project but could be worth moving them to demos. At the moment all the demo projects use pods? Perhaps they should use the built frameworks?